app/validators/multiple_episode_validator.rb

```diff
@@ -0,0 +1,7 @@
+class MultipleEpisodeValidator < ActiveModel::EachValidator
+  def validate_each(record, attribute, value)
```

Comment:
Unused method argument - `value`. If it's necessary, use `_` or `_value` as an argument name to indicate that it won't be used.